README.md

```diff
@@ -30,39 +30,54 @@ The Azure Management Libraries for .NET is a higher-level, object-oriented API f
     <td>SQL Database</td>
     <td>Databases<br>Firewalls<br>Elastic pools</td>
     <td></td>
-    <td valign="top"></td>
+    <td valign="top">More features</td>
   </tr>
   <tr>
     <td>Networking</td>
-    <td>Virtual networks<br>Network interfaces<br>IP addresses<br>Routing table<br>Network security groups<br>DNS<br>Traffic managers</td>
-    <td valign="top">Load balancers<br>Application gateways</td>
-    <td valign="top"></td>
+    <td>Virtual networks<br>Network interfaces<br>IP addresses<br>Routing table<br>Network security groups<br>Application gateways<br>DNS<br>Traffic managers</td>
+    <td valign="top">Load balancers</td>
+    <td valign="top">VPN<br>Network watchers<br>More application gateway features</td>
   </tr>
   <tr>
     <td>More services</td>
     <td>Resource Manager<br>Key Vault<br>Redis<br>CDN<br>Batch</td>
-    <td valign="top">App service - Web apps<br>Functions<br>Service bus</td>
-    <td valign="top">Monitor<br>Graph RBAC<br>DocumentDB<br>Scheduler</td>
+    <td valign="top">Web apps<br>Function Apps<br>Service bus<br>Graph RBAC<br>DocumentDB</td>
+    <td valign="top">Monitor<br>Scheduler<br>Functions management<br>Search<br>More Graph RBAC features</td>
   </tr>
   <tr>
     <td>Fundamentals</td>
-    <td>Authentication - core</td>
-    <td>Async methods</td>
+    <td>Authentication - core<br>Async methods</td>
+    <td></td>
     <td valign="top"></td>
   </tr>
 </table>
 
 > *Preview* features are flagged in documentation comments in libraries. These features are subject to change. They can be modified in any way, or even removed, in the future.
 
-**Azure Authentication**
+#### Azure Authentication
 
 The `Azure` class is the simplest entry point for creating and interacting with Azure resources.
 
 ```csharp
 IAzure azure = Azure.Authenticate(credFile).WithDefaultSubscription();
 ``` 
 
-**Create a Virtual Machine**
+#### Create a Cosmos DB with DocumentDB Programming Model
+
+You can create a Cosmos DB account by using a `define() … create()` method chain.
+
+```csharp
+var documentDBAccount = azure.DocumentDBAccounts.Define(docDBName)
+    .WithRegion(Region.USEast)
+    .WithNewResourceGroup(rgName)
+    .WithKind(DatabaseAccountKind.GlobalDocumentDB)
+    .WithSessionConsistency()
+    .WithWriteReplication(Region.USWest)
+    .WithReadReplication(Region.USCentral)
+    .Create();
+```
+
+#### Create a Virtual Machine
```

Comment:
@selvasingh Should we also add a code snippet for creating an Azure Container Registry and Azure Container Service (with Kubernetes)?